src/js/pages/Index.js

```diff
@@ -144,21 +138,34 @@ var Index = React.createClass({
     );
   },
 
-  handleWindowResize(event) {
-    const windowWidth = event.target.innerWidth;
-    const { isVisible } = this.internalStorage_get();
+  handleWindowResize() {
+    const currentWindowWidth = window.innerWidth;
 
-    if (windowWidth <= MOBILE_THRESHOLD && isVisible) {
-      SidebarActions.close();
+    if (
+      !hasViewportChanged(this.state.previousWindowWidth, currentWindowWidth)
+    ) {
+      this.setState({
```

Comment:
there is no need to update this here AFAIK 🤔

Reply:
We do need to register the state every time there is a window change.